cgd_tx_eff/src/edu/ohsu/compbio/txeff/tx_eff_hgvs.py

```diff
@@ -268,15 +269,19 @@ def _merge_annovar_with_hgvs(annovar_transcripts: list, hgvs_transcripts: list):
 
     # Iterate over every HGVS variant-transcript and see if there is a matching Annovar transcript
     for (transcript_key, hgvs_transcript) in hgvs_dict.items():
-        annovar_match = annovar_dict.get(transcript_key) or annovar_dict.get(transcript_key+'-splicing') 
-        if not annovar_match:
-            logging.debug(f"HGVS {transcript_key} does not match any Annovar merged_transcripts")
-        else:
-            logging.debug(f"Merging HGVS transcript with Annovar transcript having key {transcript_key}")
-            merged_transcripts.append(_merge(transcript_key, hgvs_transcript, annovar_match))
+        # Each annovar transcript may also exist as a splicing transcript
+        for k in [transcript_key, transcript_key + SPLICING_SUFFIX]:
+            annovar_match = annovar_dict.get(k) 
+            if not annovar_match:
+                # no need to double the number of log messages by mentioning every time a splice variant doesn't exist.
+                if not k.endswith(SPLICING_SUFFIX):
```

Comment:
Can't this nested condition just be combined with the one above for `if not annovar_match and not k.ednswith(SPLICING_SUFFIX)` ?